java/core/src/main/java/sleeper/core/statestore/transactionlog/TransactionLogStateStore.java

```diff
@@ -21,12 +21,12 @@
 public class TransactionLogStateStore extends DelegatingStateStore {
 
     public TransactionLogStateStore(Schema schema, TransactionLogStore logStore) {
```

Comment:
It doesn't look like this constructor is used any more?

Reply:
Removed, thanks